perllib/FixMyStreet/Cobrand/Buckinghamshire.pm

```diff
@@ -144,6 +144,18 @@ sub report_new_munge_before_insert {
     $report->category('Flytipping (off-road)');
 }
 
+sub post_report_sent {
+    my ($self, $report) = @_;
+
+    foreach my $body_id ($report->bodies_str_ids) {
+        if (grep $body_id, $self->_parish_ids) {
```

Comment:
This needs to be `grep { $body_id == $_ } @{$self->_parish_ids}` - this is currently always matching as $body_id is always set.

Reply:
Oh, and we want parish_bodies, not _parish_ids which are the area IDs.